syntax/Cargo.toml

```diff
@@ -0,0 +1,71 @@
+[package]
+name = "miden-assembly-syntax"
+version = "0.15.0"
+description = "Parsing and semantic analysis of the Miden Assembly language"
+documentation = "https://docs.rs/miden-assembly-syntax/0.15.0"
+readme = "README.md"
+categories = ["compilers", "no-std"]
+keywords = ["assembly", "language", "syntax", "miden"]
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[features]
+default = ["std"]
+std = [
+    "aho-corasick/std",
+    "miette/fancy",
+    "miette/std",
+    "miden-core/std",
+    "thiserror/std",
+]
+arbitrary = ["dep:proptest", "dep:proptest-derive"]
```

Comment:
Why `arbitrary` as a name?

Reply:
That's the convention I've seen in the ecosystem for depending on the `proptest` crates in order to implement the `Arbitrary` trait for types as an opt-in feature, so chose to use it here as well.